tools/cldr-apps/js/src/views/WaitingPanel.vue

```diff
@@ -122,6 +121,14 @@ export default {
                   }),
                 4000
               );
+              run()
+              .catch(e => {
+                // We can get here if run() was not able to boot the page
+                // That's OK, it may have been a waiting page. Reload should
+                // clear it up.
+                console.log(`run() threw an error, so we will do a page reload. Err was: ${e}`);
```

Comment:
Question; if there is a persistent problem loading, will this go into an infinite loop?

Reply:
@macchiati before getting to this line, the front end checks with the server to make sure that it's ready to start up.  The only error here is that it wasn't able to start the surveytool front end _without reloading the page_.  If a problem happened after reload, it should be caught by the other err handlers that this PR puts in.